dataland-frontend/src/components/forms/parts/elements/derived/ActivitySelector.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <UploadFormSubcategoryHeader label="Activity Name" description="Name of the activity." />
+  <div class="form-field next-to-each-other">
+    <p>
+      <b> {{ selectedActivities ? selectedActivities.name : "" }}</b>
+    </p>
+
+    <FormKit
+      type="hidden"
+      name="activityName"
+      :modelValue="selectedActivities ? selectedActivities.value : ''"
+      disabled="true"
+    />
+
+    <PrimeButton
+      data-test="dataTestChooseActivityButton"
+      :label="selectedActivities ? 'Change Activity' : 'Choose Activity'"
+      class="p-button-text p-0 m-0"
+      :icon="selectedActivities ? 'pi pi-pencil' : 'pi pi-list'"
+      @focus="inputFocused"
```

Comment:
Why open the tree on focus and not on click?

Reply:
Why on click and not on focus? :) On focus, clicking through the input by tab will open the tree, but I don't know if it's important to us. I guess the choice was random.